src/contexts/Balance/Provider.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { Provider } from '@ethersproject/abstract-provider';
+
+// Constants
+import { WPOKT_ADDRESS } from 'constants/index';
+
+// Context
+import { Web3Context } from 'contexts/Web3';
+import { BalanceContext } from './Context';
+
+// Utils
+import { getERC20Contract } from 'utils';
+
+export const BalanceProvider: React.FC = ({ children }) => {
+  const { address, provider } = React.useContext(Web3Context);
+  const [wpoktBalance, setWpoktBalance] = React.useState<string>();
+
+  const getBalance = async (provider: Provider, tokenAddress: string, userAddress: string): Promise<string> => {
+    const tokenContract = getERC20Contract(provider, tokenAddress);
+    try {
+      const balance = await tokenContract.balanceOf(userAddress);
+      return balance.toString();
+    } catch (e) {
+      console.error(e);
+      return '0';
+    }
+  };
+
+  const fetchBalances = React.useCallback(
```

Comment:
also, just to double check: are we calling every `React`  import with the dot syntax instead of importing it? (e.g `React.useEffect`) It's fine if we're doing so, I like it—But let's double-check make sure it's like that on the whole codebase 💯

Reply:
Yup, just double-checked, and it's consistent throughout!